packages/dx-react-chart-bootstrap4/src/templates/tooltip/overlay.jsx

```diff
@@ -20,9 +20,7 @@ export class Overlay extends React.PureComponent {
         modifiers={popperModifiers}
         {...restProps}
       >
-        <div className="popover-body">
-          {children}
-        </div>
+        {children}
```

Comment:
bs4 and mui files are not the same. Where is `arrowComponent` here? and it missing in the prop validation

Reply:
The `arrowComponent` in the popover.jsx